### PR TITLE
Blog post about SymPy 1.4

### PR DESCRIPTION
**Pull request by asmeurer**
The post date here is provisional and should be updated before merging.

This will be cross-posted on my blog. The draft version on my blog can be seen
at https://www.asmeurer.com/blog/_sympy-1.4/posts/whats-new-in-sympy-14/.

~~I still need to add one sentence blurbs on what Quansight and Quansight Labs
are.~~ Done.

### CODE REVIEW

**Comment by asmeurer** [comment]
This is ready for review.

**Comment by asmeurer** [comment]
The CircleCI build should be made to not fail on pull requests.

**Comment by hameerabbasi** [comment]
@asmeurer For security reasons it doesn't run if someone outside the Quansight-Labs org starts a build/PR. :-)

**Comment by asmeurer** [comment]
Is it possible to run the build but not try to upload? It should do that anyway even for builds on the main repo if the branch isn't master, no? 

Also I don't know how Circle works, but on Travis secure environment variables aren't accessible from builds from forks, so it isn't a concern.

**Comment by hameerabbasi** [comment]
We're using Circle because Travis has no artifacts... And we could (I guess), but we'd have to separate the workflows and envs, copy data over from one env to another. It's a hassle.

**Comment by asmeurer** [comment]
It's not too hard with Travis using [doctr](https://drdoctr.github.io) (a tool I wrote btw). But I don't really care which is used, especially if someone else is maintaining it. It should be possible to do only the build with any system on PR builds from forks.

**Comment by asmeurer** [comment]
Looking for comments on the text here, especially the paragraph at the top about Quansight. The cross-post on my blog has an extra sentence describing what Quansight is, but other than that the two posts are identical.

**Comment by asmeurer** [comment]
Oh also, can/should I add a link to the top to the cross post on my blog (I plan to have a link at the top of my blog linking to the cross-post here)?

**Comment by rgommers** [comment]
> It's not too hard with Travis using doctr (a tool I wrote btw).

That will come in handy at some point, thanks for pointing it out @asmeurer

File: posts/2019/04/whats-new-in-sympy-14.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
+<!--
+.. title: What's New in SymPy 1.4
+.. slug: whats-new-in-sympy-14
+.. date: 2019-04-29
+.. author: Aaron Meurer
+.. tags: sympy, Labs, mathjax
```
Comment (rgommers) [inline posts/2019/04/whats-new-in-sympy-14.md]: the post is unrelated to mathjax right? if so, I would remove the tag. Also, spell it SymPy?

File: posts/2019/04/whats-new-in-sympy-14.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+While I did not personally work on any of the changes listed below (my work
+for this release tended to be more invisible, behind the scenes fixes), I did
+do the release itself.
+
+# Automatic LaTeX rendering in the Jupyter notebook
+
+Prior to SymPy 1.4, SymPy expressions in the notebook rendered by default with their
+ string representation. To get `LaTeX` output, you had to call `init_printing()`:
+
+<img src="../../../../images/sympy-1.3-notebook.png" alt="SymPy 1.3 rendering in the Jupyter lab notebook">
```
Comment (rgommers) [inline posts/2019/04/whats-new-in-sympy-14.md]: These relative links will be fragile I think. Better to use paths starting from the deployed root and use the markdown syntax:
```
![alt text](/images/filename.png)`

also, for my post I put images in `images/` not `files/images/`. I believe `files/` is for things that should be put in `/`, post images would be good to have in one place.

File: posts/2019/04/whats-new-in-sympy-14.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+[Quansight](https://www.quansight.com/), under the heading of [Quansight
+Labs](https://www.quansight.com/labs). Quansight Labs is a public-benefit
+division of Quansight. It provides a home for a "PyData Core Team" which
+consists of developers, community managers, designers, and documentation
+writers who build open-source technology and grow open-source communities
+around all aspects of the AI and Data Science workflow. As a part of this, I
+am able to spend a fraction of my time working on SymPy.
+[SymPy](https://www.sympy.org/en/index.html), for those who do not know, is a
+symbolic mathematics library written in pure Python. I am the lead maintainer
+of SymPy.
```
Comment (rgommers) [inline posts/2019/04/whats-new-in-sympy-14.md]: Would be good to have others have a look at this (I already saw it before).

File: posts/2019/04/whats-new-in-sympy-14.md (hunk 1 above)
Comment (asmeurer) [reply posts/2019/04/whats-new-in-sympy-14.md]: I'm unclear if the `mathjax` tag or `has_math` enables MathJax. Note that the `mathjax` tag is automatically hidden due to a setting in conf.py.

Comment (asmeurer) [reply posts/2019/04/whats-new-in-sympy-14.md]: I guess it still works if I remove the tag.

**Comment by rgommers** [comment]
Hmm something just changed, CI is no longer happy:
```
An unauthorized user attempted to run this workflow. If you have authorization,
click Rerun button to rerun the workflow.
```
I clicked re-run but that resulted in the same.

**Comment by hameerabbasi** [comment]
I re-ran it. AFAIK you need to follow the project on CircleCI and be in the Quansight-Labs org.

**Comment by rgommers** [comment]
> I re-ran it. AFAIK you need to follow the project on CircleCI and be in the Quansight-Labs org.

That should not be needed. For numpy and scipy for example it works just fine on any PR.

**Comment by hameerabbasi** [comment]
The issue here is that we have deployment in the same PR... And if the token gets compromised, anyone can impersonate me on GitHub. 🙂 

The NumPy/SciPy CIs are just tests, no secure environment variables.

**Comment by rgommers** [comment]
No there's deployment to GitHub Pages as well on merge to master. https://docs.scipy.org/doc/scipy-dev/reference/ is http://scipy.github.io/devdocs/

**Comment by hameerabbasi** [comment]
If we merge `master` into this, the issue should be fixed. 🙂

**Comment by rgommers** [comment]
CI now works too (thanks Hameer), let's get this up. I updated the publication data and added the teaser markdown comment so it shows only the first section on the main page followed by "Read more ...."

Thanks Aaron!

**Pull request merged**